app/Http/Controllers/UserController.php

```diff
@@ -2,50 +2,191 @@
 
 namespace Suyabay\Http\Controllers;
 
+use Validator;
+use Hash;
 use Suyabay\User;
 use Suyabay\Role;
 use Suyabay\Invite;
 use Suyabay\Http\Requests;
 use Illuminate\Http\Request;
-use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Response;
 use Illuminate\Database\QueryException;
 use Suyabay\Http\Controllers\Controller;
+use League\Fractal\Manager;
+use League\Fractal\Resource\Collection;
+use Suyabay\Http\Transformers\UserTransformer;
```

Comment:
**Code Quality - Nit pick**
Make sure the importations sure arranged from shortest to longest.